backend/editor/entries.py

```diff
@@ -270,20 +270,30 @@ async def list_projects(self, status=None):
             "MATCH (n:PROJECT)",
             "OPTIONAL MATCH (error_node:ERRORS {branch_name: n.branch_name, id: n.id})",
         ]
+
         params = {}
         if status is not None:
             # List only projects matching status
             query.append("WHERE n.status = $status")
             params["status"] = status
+
         query.extend(
             [
                 "WITH n, size(error_node.errors) AS errors_count",
                 "RETURN n{.*, errors_count: errors_count}",
                 "ORDER BY n.created_at",
             ]
         )
-        result = await get_current_transaction().run("\n".join(query), params)
-        return await async_list(result)
+
+        queryResult = await get_current_transaction().run("\n".join(query), params)
+
+        result = []
+
+        async for queryResultList in queryResult:
+            for item in queryResultList:
+                result.append(item)
+
```

Comment:
@alexgarel @nobeeakon Do you think there's a better way to do this? I'd probably like to change the neo4j query to remove the nested lists.

Reply:
Couldn't figure out how to do this in Neo4j, but if we decide to do it in python, this can flatter the array:
  ```python
  result = await get_current_transaction().run(" ".join(query), params)
  return [record["n"] for record in await result.data()]
```

Reply:
It's not a big problem to unnest lists in the code. API readability is more important, it's quite fast to do this in python.